scripts/tools/generate-integrations-json.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+import os
+import re
+from glob import glob
+
+import yaml
+
+import airflow
+
+if __name__ != "__main__":
+    raise Exception(
+        "This file is intended to be executed as an executable program. You cannot use it as a module."
+        "To run this script, run the './generate-integrations-json.py >integrations.json' command"
```

Comment:
The output filename should not be customized as this is a source of potential typos.

Reply:
Ok, I'll take the approach in https://github.com/apache/airflow/blob/master/docs/publish_docs.py and just take `AIRFLOW_SITE_DIR = os.environ.get('AIRFLOW_SITE_DIRECTORY')` and output directly to a file in that location

Reply:
done